src/xERC20/contracts/XUltraLRT.sol

```diff
@@ -133,14 +133,52 @@ contract XUltraLRT is
         Message memory message = abi.decode(_message, (Message));
         // handle message
         if (message.msgType == MSG_TYPE.MINT) {
-            // TODO: introduce limits use burn with caller
-            _mint(message.sender, message.amount);
+            _handleCrossChainMint(message.sender, message.amount);
         } else if (message.msgType == MSG_TYPE.BURN) {
```

Comment:
should we just remove this if not used?

Reply:
keeping it, have plan to 2 way mint & burn.